src/main/java/bio/terra/service/auth/iam/sam/SamIam.java

```diff
@@ -567,27 +564,23 @@ public RepositoryStatusModelSystems samStatus() {
   @Override
   public UserStatus registerUser(String accessToken) throws InterruptedException {
     logger.info("Registering the ingest service account into Terra");
-    SamRetry.retry(
+    return SamRetry.retry(
```

Comment:
It looks like the return value of `registerUser()` is never used. I would change this to return `void` instead, especially now that it can return `null`.